src/pages/Transfer/CrossChainTransferForm/components/ChainInputs/ChainInputs.styles.tsx

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+const IconWrapper = styled('div')`
+  margin-top: 1.75rem;
```

Comment:
I haven't put this in the theme file because it won't be reused—it's only here because we need to align the icon (`alignItems='center'` centres the icon in relation to the combined label/input which isn't what we need)